homeassistant/components/notify/telegram.py

```diff
@@ -66,49 +66,49 @@ def send_message(self, message="", **kwargs):
         title = kwargs.get(ATTR_TITLE)
         data = kwargs.get(ATTR_DATA, {})
 
-        # send message
         try:
             self.bot.sendMessage(chat_id=self._chat_id,
-                                 text=title + "  " + message)
+                                 text='{} {}'.format(title, message))
         except telegram.error.TelegramError:
-            _LOGGER.exception("Error sending message.")
+            _LOGGER.exception("Error sending message")
             return
 
-        # send photo
-        if ATTR_PHOTO in data:
-            # if not a list
+        try:
```

Comment:
Instead of a `try`, just avoiding the exception would be cleaner:
```python
if not data:
    return
photos = data[ATTR_PHOTO]

if not isinstance(photos, list):
    photos = [photos]
```